airflow/providers/microsoft/azure/provider.yaml

```
@@ -59,6 +59,10 @@ integrations:
     external-doc-url: https://cloud.google.com/storage/
     logo: /integration-logos/azure/Microsoft-Azure-Fileshare.png
     tags: [azure]
+  - integration-name: Microsoft Azure Data Factory
+    external-doc-url: https://azure.microsoft.com/en-us/services/data-factory/
+    logo: /integration-logos/azure/Azure Data Factory.svg
+    tags: [azure] 
```

```suggestion
    tags: [azure]
```

Static check is failing because of whitespace